Description: Download the correct version for arm64
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 burpsuite (2024.2.1.3-1parrot3) stable; urgency=medium
 .
   * Add debian patches for packaging
   * Remove openjdk in runtime dependency
   * Add Java JRE to build time dependency
   * Try making folder burpsuite with write permission
   * New patch to fix build crash
Author: Nong Hoang Tu <dmknght@parrotsec.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2024-10-28

--- burpsuite-2024.2.1.3.orig/Makefile
+++ burpsuite-2024.2.1.3/Makefile
@@ -1,2 +1,12 @@
+ARCH := $(shell uname -m)
+ifeq ($(ARCH), aarch64)
+	url="https://portswigger-cdn.net/burp/releases/download?product=community&type=LinuxArm64"
+else
+	url="https://portswigger-cdn.net/burp/releases/download?product=community&type=Linux"
+endif
 install:
-	wget -q "https://portswigger-cdn.net/burp/releases/download?product=community&type=Jar" -O burpsuite.jar
+	wget -q $(url) -O burpsuite.sh
+	chmod +x burpsuite.sh
+	mkdir -p burpsuite
+	chmod +w burpsuite
+	./burpsuite.sh -q -dir burpsuite
--- burpsuite-2024.2.1.3.orig/burpsuite
+++ /dev/null
@@ -1,3 +0,0 @@
-#!/bin/bash
-
-/usr/lib/jvm/java-17-openjdk-$(dpkg-architecture -q DEB_HOST_ARCH)/bin/java -jar -Xmx4g --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.desktop/javax.swing=ALL-UNNAMED /usr/share/burpsuite/burpsuite.jar
